### DISCUSSION TITLE
How can I highlight my particular commits on GitHub?

### DISCUSSION THREAD

**Discussion by AakarshitAgarwal** [github/india#958] [GitHub Platform]
We sometimes use GitHub to backup our codes in private/public repos
If I or someone is reviewing my repos again how can I find out the important commits which I did?
Is there any way I can highlight my commits on GitHub so important commit/code snippets can be recognized while reviewing?

**Comment by FRTMentor** [comment]
GitHub commits are well maintained from the GitHub community.
No need to reorganized or to be highlighted.
In case want research or know more on that features please check with GitHub community.
